tests/meta/archive/test_archive_api.py

```diff
@@ -214,10 +221,160 @@ def test_archive_all_non_archived_spring_files(
             )
             if sample and sample.archive_location == ArchiveLocations.KAROLINSKA_BUCKET:
                 assert file.archive
-            else:
-                assert not file.archive
-        else:
-            assert not file.archive
+
+
+def test_get_archival_status_done(
+    spring_archive_api: SpringArchiveAPI,
+    caplog,
+    ok_ddn_job_status_response,
+    archive_request_json,
+    header_with_test_auth_token,
+):
+    # GIVEN a file with an ongoing archival
+    file: File = spring_archive_api.housekeeper_api.files().first()
+    spring_archive_api.housekeeper_api.add_archives(files=[Path(file.path)], archive_task_id=123)
+
+    # WHEN querying the task id and getting a "COMPLETED" response
+    with mock.patch.object(
+        AuthToken,
+        "model_validate",
+        return_value=AuthToken(
+            access="test_auth_token",
+            expire=int((datetime.now() + timedelta(minutes=20)).timestamp()),
+            refresh="test_refresh_token",
+        ),
+    ), mock.patch.object(
+        APIRequest,
+        "api_request_from_content",
+        return_value=ok_ddn_job_status_response,
+    ), mock.patch.object(
+        GetJobStatusPayload,
```

Comment:
Maybe you could turn these two objects into one or several fixtures so that it doesn't appear so much code as duplicated